openhsi/data.py

```diff
@@ -465,7 +466,10 @@ def save(self, save_dir:str, preconfig_meta_path:str=None, prefix:str="", suffix
         self.nc.datacube.attrs["description"] = "hyperspectral datacube"
 
         self.nc.to_netcdf(f"{self.directory}/{prefix}{self.timestamps[0].strftime('%Y_%m_%d-%H_%M_%S')}{suffix}.nc")
-        hv.save(self.show("matplotlib",robust=True),f"{self.directory}/{prefix}{self.timestamps[0].strftime('%Y_%m_%d-%H_%M_%S')}{suffix}.png")
+
+        fig = self.show("matplotlib",hist_eq=True,quick_imshow=True)
```

Comment:
should have flag on whether to actually show the figure, i.e. remote operation.

Reply:
I think this is fine as is. It returns a figure object so it doesn't actually show the figure by default.
In Jupyter, each cell shows you the last returned object and in this case it show() shows the figure.
It's consistent with matplotlib. 

In this method .save(), I use the returned figure object for saving a .png version. Do you mean having a flag to save the datacube but not the RGB pic with it?

Reply:
right, I originally read that as matplotlib/hv `show`. perhaps the `DataCube.show(...)` should have a different name to avoid confusion? makeFigure for example?

Reply:
Or `imshow` to match the matplotlib implementation. The thing is, holoviews works quite differently from matplotlib even though it can use the matplotlib backend. It can make matplotlib figures but they are contained inside holoviews objects so it's composable and works the same way if using bokeh. For example, this is how you would view a holoviews figure outside of a notebook (https://holoviews.org/user_guide/Plots_and_Renderers.html).
```python
from IPython.display import display_png
png, info = renderer(curve, fmt='png') # curve is a holoviews figure
display_png(png, raw=True)
```
There is no `hv.show` so this is a unique method that doesn't contradict. However, I can understand your confusion with it mixing with `plt.show`. `make_figure` seems clunky compared to `show` and I haven't decided if there's a better way yet.

Reply:
no worries, no need to change now, just an observation.